src/app/tests/suites/certification/Test_TC_CNET_4_13.yaml

```diff
@@ -122,20 +123,28 @@ tests:
           ( Pre-Condition)
       disabled: true
 
+    - label: "Step 7: TH sends the CommissioningComplete command to the DUT"
+      verification: |
+          ./chip-tool generalcommissioning CommissioningComplete 1 0
+
+          The test case is not verifiable in RPI platform. As MaxNetworks value is 1 but expected is 4
+          ( Pre-Condition)
+      disabled: true
```

Comment:
This is a new disabled (i.e. manual) test step. We generally do not allow new manual test steps anymore.

Reply:
Thanks for the feedback. The change done here is allow this test to align with the matter spec. Since the test was a manual test and has not been changed to an automated/semi automated test what would be the correct course of action to get it to align with the spec?

Reply:
@andy31415 since this is a old cert test, we can make updates to the manual test. 
@Kshitijjain21 But we can not merge this change unless test plan PR is approved.

Reply:
This adds several new steps that are manual ... so it kind of makes the problem worse. I understand the justification of "we add a test step in a long list of test steps that is already manual" however I also am concerned that this does increase the count of manual stuff invocation. We should not do that.